docs/about/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: About
+layout: page
+---
+
+# About JHOVE
+
+JHOVE is currently maintained by the [Open Preservation Foundation](http://openpreservation.org). Development effort is guided by the JHOVE Product Board, comprising OPF members and software supporters.
+
+The original JHOVE project was a collaboration between [JSTOR](http://www.jstor.org) and [Harvard University Library](http://hul.harvard.edu/) with funding from the [Andrew W. Mellon Foundation](http://www.mellon.org) for the Electronic-Archiving Initiative.
+
+{: #support-jhove .section-heading}
+## Support JHOVE
+
+JHOVE is one of the most widely-used digital preservation tools in the community, and it is also used as a component in several commercial digital preservation systems. It is open source, meaning it is free to download, use, and modify. However, it is not free of costs to host, maintain, support, or develop the software. Currently, only OPF members support JHOVE as part of their membership fees.
+
+### So, how can you help?
+
+#### Become an OPF member
+
+By becoming a member of the OPF, organisations are directly helping to fund the hosting, maintenance, and development of the software, ensuring it is robust and reliable.
+
+Members can also choose to guide the roadmap and:
+- Sit on the JHOVE Product Board
+- Vote for bug fixes and enhancements to be included in each release cycle
+- Receive technical support
+
+Find out more: [http://openpreservation.org/about/join/](http://openpreservation.org/about/join/)
```

Comment:
If the link text is just the link then it may be better to use `<link>` instead, see <https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>